src/core_plugins/kibana/public/discover/styles/main.less

```diff
@@ -2,7 +2,7 @@
 @import "~ui/styles/local_search.less";
 
 .tab-discover {
-  overflow-x: hidden;
+  overflow: hidden;
```

Comment:
Can you describe what this is achieving? Best I can tell this doesn't do anything, the scrollbar actually comes from the `.app-wrapper-panel`.

Reply:
Turns out that `overflow-x` results in a vertical scrollbar showing up:

![image](https://user-images.githubusercontent.com/1238659/37746066-96ea625c-2d35-11e8-93ff-7d837607621c.png)

Clipping both axes with `overflow` fixes it:

![image](https://user-images.githubusercontent.com/1238659/37746079-9fcd3bc4-2d35-11e8-8282-9d906178dc1d.png)